app/controllers/provider_interface/user_invitation/base_controller.rb

```diff
@@ -0,0 +1,25 @@
+module ProviderInterface
+  module UserInvitation
+    class BaseController < ProviderInterfaceController
```

Comment:
The plan here is for the permissions controller of this flow to also inherit from this base controller and have new & create actions